calour/analysis.py

```diff
@@ -160,9 +165,18 @@ def diff_abundance(exp: Experiment, field, val1, val2=None, method='meandiff', t
     Returns
     -------
     Experiment
-        A new experiment with only significant (FDR <= maxfval) difference, sorted according to difference
+        A new experiment with only significant (FDR <= maxfval) difference, sorted according to the effect size.
 
+        The new experiment contains additional feature_metadata_fields that include:
+            '_calour_diff_abundance_pval' : the p-value for the feature
```

Comment:
make these a list and delete empty lines

Reply:
is this what you meant?